app/src/pages/CountersPage/containers/CountersPageContainer.js

```diff
@@ -0,0 +1,85 @@
+import {useState, useCallback, useMemo} from "react";
+import CounterView from "../../../commonComponents/CounterView";
+
+
+import CountersPageLayout from "../components/CountersPageLayout/index";
+
+const CountersPageContainers = () => {
+   const [countersList, setCountersList] = useState([])  
+   
+   const handleAddCounter = useCallback(() => { 
+      const UpdateCounter = countersList.map((counter) => ({
```

Comment:
1) the names of variables in JS should start with lower case letter:) 
2) in your variable you store the array of items, that's why the name should be plural - `updatedCounters`, 's' in the end